src/api/Metadata.js

```diff
@@ -257,6 +267,58 @@ class Metadata extends File {
         }
     }
 
+    /**
+     * Gets classification for a file.
+     *
+     * @param {BoxItem} file - File object for which we are getting classification
+     * @param {Function} successCallback - Success callback
+     * @param {Function} errorCallback - Error callback
+     * @param {boolean} forceFetch - True to clear the cache and make an api call
+     * @return {Promise}
+     */
+    async getClassification(
+        file: BoxItem,
```

Comment:
Since you are not using the file object anymore to get classification out, get rid of it from the SIDEBAR PROPERTIES TO FETCH. In this way metadata (at least this one) is isolated doesn't break the files API call